src/rqt_plot/data_plot/pyqtgraph_data_plot.py

```diff
@@ -43,7 +43,12 @@
         def parse_version(s):
             return [int(x) for x in re.sub(r'(\.0+)*$', '', s).split('.')]
 
-    from pyqtgraph import __version__ as pyqtgraph_version
+    try:
+        from pyqtgraph import __version__ as pyqtgraph_version
+    except:
```

Comment:
Please catch a specific exception type here as well as add a comment when it is expected to be raised.

Reply:
@dirk-thomas sorry for late, updated to catch only "RuntimeError"